routers/stream_router.py

```
@@ -1,11 +1,25 @@
-from fastapi import APIRouter
+from fastapi import APIRouter,HTTPException
+from schemas.stream_schema import *;
```

Ruff (0.15.9)

[error] `from schemas.stream_schema import *` used; unable to detect undefined names

(F403)